### PR TITLE
[FIXED JENKINS-38047] Allow multiple positional parameters.

### PR DESCRIPTION
**Pull request by abayer**
[JENKINS-38047](https://issues.jenkins-ci.org/browse/JENKINS-38047)

Phew. That was kind of nutty.

cc @reviewbybees esp @jglick @michaelneale @oleg-nenashev @kohsuke

### CODE REVIEW

**Comment by abayer** [comment]
Oops. Pushed with gobs of debug crap still in. Will clean up shortly.

**Comment by abayer** [comment]
Aaaaand cleaned up. =)

File: src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/BasicModelDefTest.java [hunk 1]
```diff
@@ -150,13 +150,13 @@ public void globalLibrarySuccess() throws Exception {
 
         j.assertLogContains("[nothing here]", b);
         j.assertLogContains("map call(1,2)", b);
-        /* TODO called the wrong way: closure1([a:1])
+        /* TODO called the wrong way: closure1([a:1]) */
```
Comment (jglick) [inline src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/BasicModelDefTest.java]: :ant: delete the obsolete comment

File: src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/BasicModelDefTest.java [hunk 2]
```diff
@@ -150,13 +150,13 @@ public void globalLibrarySuccess() throws Exception {
...
         j.assertLogContains("closure1(1)", b);
-        */
+
         j.assertLogContains("running inside closure1", b);
-        /* TODO silently ignored
+        /* TODO silently ignored */
```
Comment (jglick) [inline src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/BasicModelDefTest.java]: :ant: ditto

**Comment by jglick** [comment]
:bee: with scant comprehension

**Pull request merged**